sklearn/utils/multiclass.py

```diff
@@ -344,11 +344,33 @@ def _check_partial_fit_first_call(clf, classes=None):
 
         else:
             # This is the first call to partial_fit
+            if issparse(y):
+                raise ValueError("Sparse y is not supported.")
+
+            # Validate classes against unique labels in y
```

Comment:
Adding partial fit to trees is not something that will happen any time soon, as it would require an entirely different approach afaik. And I'm not sure if adding multi-output support to sklearn was a good idea.